Problems/2309/ans_2309_BNR.py

```diff
@@ -0,0 +1,27 @@
+#https://www.acmicpc.net/problem/2309
+
+
+P = []
+array = []
+copyarray = []
+result = []
+for _ in range(9):
+    P.append(int(input()))
+
+def find():
+    for i in range(len(P)):
+        array = P.copy()
+        array[i]=0
+        for j in range(i+1,len(P)):
+            copyarray = array.copy()
+            # print(copyarray)
+            copyarray[j]=0
+            if sum(copyarray) == 100:
+                # print(copyarray)
+                return copyarray
```

Comment:
```suggestion
    for i in range(len(P)):
        for j in range(i+1,len(P)):
            if sum(P)-P[i]-P[j] == 100:
                P.remove(j)
                P.remove(i)
                return P
```
너무 어렵게 생각했지 싶다 ㅋㅋ
해당 함수 return 형 유지하기위해서 remove를 사용함. j가 i보다 먼저 빠져야함에 주의.

Reply:
이 간단한걸 어렵게 풀었네요ㅋㅋㅋ